sdk/bare/common/sys/Logging-Binary-Format.md

```diff
@@ -0,0 +1,40 @@
+# Logging Binary Format
+
+## Overview
+
+The AMDC can dump the logged variables to the UART serial console using either text-based formatting or the binary
+format. This document describes the binary formatting which appears on the UART interface.
+
+## Dumping Variables
+
+Each logged variable must be dumped one at a time (i.e. if the user has 10 slots, each must be dumped one by one).
+
+To start a dump of the Nth variable slot: `log dump bin N`
+
+## Binary Format
+
+All data is little-endian.
+
+### Magic Header (16 bytes)
+
+0x12345678 (repeated four times)
+
+### Number of Samples (4 bytes, integer)
+
+0x00000010
```

Comment:
> 0x00000010

I am not sure how to interpret this number. Is this an example for a value that this field could have?

Reply:
Yes, that was a crude example. I have updated the docs to be more clear here: 1748bc20a2ed5a62a42d35443b283ab484900325